modules/identity-provider-keystone-about.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-keystone-identity-provider.adoc
+
+[id="identity-provider-keystone-about_{context}"]
+= About Keystone authentication
+
+http://docs.openstack.org/developer/keystone/[Keystone] is an OpenStack project that provides identity, token, catalog, and policy services.
+
+You can configure the integration with Keystone so that the new {product-title} users are based on either the Keystone user names or unique Keystone IDs. With both methods, users log in by entering their Keystone user name and password. Basing the {product-title} users off of the Keystone ID is more secure. If you delete a Keystone user and create a new Keystone user with that user name, the new user might have access to the old user's resources.
```

Comment:
First sentence users are based **_on_**.  Third sentence users are based _**off**_.   Should these be the same, from a grammatical perspective? (Nominee for the smallest nit of the year award....)

Reply:
Does the lase sentence need a lead in, such as _Because_ or _This is because_? It took me a moment to realize the sentence was describing why Keystone ID is more secure (right??).